features/FEATURE_COMMON_PAL/mbed-trace/mbed-trace/mbed_trace.h

```diff
@@ -297,10 +298,8 @@ const char* mbed_trace_include_filters_get(void);
  * @param fmt    trace format (like printf)
  * @param ...    variable arguments related to fmt
  */
-#if defined(__GNUC__) || defined(__CC_ARM)
-void mbed_tracef(uint8_t dlevel, const char* grp, const char *fmt, ...) __attribute__ ((__format__(__printf__, 3, 4)));
```

Comment:
Why are the format argument checks being removed? These are really useful to have.

Reply:
Format argument check is added https://github.com/ARMmbed/mbed-os/pull/5965/files#diff-b5d7ec54285660451d0d038432db3997R94